src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -2156,3 +2190,178 @@
     text: >
         az tag update --resource-id /subscriptions/{sub-id}/resourcegroups/{rg}/providers/Microsoft.Compute/virtualMachines/{vmName} --operation delete --tags key1=value1
 """
+
+helps['ts'] = """
+type: group
+short-summary: Manage template specs at subscription or resource group scope.
+"""
+
+helps['ts create'] = """
+type: command
+short-summary: Create a template spec and or template spec version.
+parameters:
+  - name: --resource-group -g
+    type: string
+    short-summary: Name of resource group.
+  - name: --name -n
+    type: string
+    short-summary: Name of the template spec.
+  - name: --version -v
+    type: string
+    short-summary: The template spec version.
+  - name: --template-file -f
+    type: template_spec
+    short-summary: The path to the template file.
+  - name: --location -l
+    short-summary: The location to store the template-spec and template-spec version(s). Cannot be changed after creation.
+  - name: --display-name -d
+    type: string
+    short-summary: The display name of the template spec.
+  - name : --description
+    type: string
+    short-summary: The description of the root template spec.
+  - name : --version-description
+    type: string
+    short-summary: The description of template spec version.
```

Comment:
Could we move these parameters description to _params.py and reuse some common params in `az ts` command group? 
Example:
https://github.com/Azure/azure-cli/blob/f67024b48bd0efea4084f028897d7b7a7abf35c9/src/azure-cli/azure/cli/command_modules/storage/_params.py#L176-L186

Reply:
Done.